b2/console_tool.py

```diff
@@ -767,6 +769,11 @@ class Sync(Command):
 
         Note that --includeRegex cannot be used without --excludeRegex.
 
+        You can specify --excludeDirRegex to selectively ignore whole directories
+        that match the given pattern. There is not other option, which overrides
+        --excludeDirRegex exclusions. The pattern is a regular expression
```

Comment:
Instead of "There is not other option, which overrides --excludeDirRegex exclusions", how about:

```
When a directory is excluded, all of the files within it are excluded, even if they match an --includeRegex pattern.
```